drivers/i2c/busses/Kconfig

```diff
@@ -390,6 +390,17 @@ config I2C_ALTERA
 	  This driver can also be built as a module.  If so, the module
 	  will be called i2c-altera.
 
+config I2C_AST2600
```

Comment:
ast2600/ast2700 enabled for compatible = "aspeed,ast2600-i2cv2"; are you planing to add this in aspeed-g6.dtsi ?

Reply:
Right now, enabling for SP7 only. Will test and add sp5/2600 drivers later

Reply:
Ack